_includes/assets/sass/_components.scss

```diff
@@ -274,4 +274,44 @@ COMPONENTS
 			width: govuk-px-to-rem(64px);
 		}
 	}
+}
+
+// Breadcrumbs block
+
+// Uses govuk sass and replica markup (found in breadcrumbs.njk) as a base, then we override cosmetic styles; color, text-decoration etc.
+@import "node_modules/govuk-frontend/govuk/components/breadcrumbs/breadcrumbs";
+
+.breadcrumbs-block {
+	border-top: 1px solid $grey-border;
+
+	.govuk-breadcrumbs {
+		font-family: $primary-font;
+	}
+
+	.govuk-breadcrumbs__link {
+		font-family: $primary-font;
+		color: $red;
```

Comment:
Instead of writing out the CSS to override it, you can set the variable that is used by govuk beforehand, the one for link text is here: https://github.com/alphagov/govuk-frontend/blob/899264cd89559d31f0a790682254d94ecf66418b/package/govuk/settings/_colours-applied.scss#L128

Reply:
Great, amended.